src/components/PrivateRoute/PrivateRoute.jsx

```diff
@@ -6,6 +6,19 @@ import { useUser } from '../../context/UserContext';
 export default function PrivateRoute({ children, ...rest }) {
   // TODO: Use the user in context to determine whether to redirect to /login
   // const { user } = useUser();
+  const { user } = useUser();
 
```

Comment:
nice work here creating your private route to add to the application to check if there is a user.